subgraphs/alpaca-finance-lending/src/utils/constants.ts

```diff
@@ -12,9 +13,9 @@ export namespace PositionSide {
 export const PROTOCOL_ID = "Alpaca Lending";
 export const PROTOCOL_NAME = "Alpaca Finance Lending Protocol";
 export const PROTOCOL_SLUG = "alpaca-finance-lending";
-export const PROTOCOL_SCHEMA_VERSION = "2.0.1";
-export const PROTOCOL_SUBGRAPH_VERSION = "1.0.0";
-export const PROTOCOL_METHODOLOGY_VERSION = "1.0.0";
+export const PROTOCOL_SCHEMA_VERSION = Versions.getSchemaVersion();
```

Comment:
https://github.com/messari/subgraphs/blob/master/subgraphs/aave-v2-forks/src/helpers.ts#L94-L96 could you add this to it?